src/selectors/entities/posts.ts

```diff
@@ -366,6 +367,29 @@ export function makeGetPostsForThread(): (state: GlobalState, props: {rootId: $I
     );
 }
 
+// The selector below filters current user if it exists. Excluding currentUser is just for convinience
+export function makeGetProfilesForThread(): (state: GlobalState, props: {rootId: $ID<Post>}) => Array<UserProfile> {
```

Comment:
0/5 on this, but I don't think we necessarily need to keep passing the arguments into these selectors as objects. I think reselect intended you to pass component props directly into the selector, but that feels awkward to me.

Reply:
i am not sure i understand. I could avoid even defining them, but TS is complaining about it 🤷‍♂️

Reply:
TS wants them because it's passed into `getPostsForThread`, but I think if you define the selector like
```js
function makeGetSomething(): (state: GlobalState, rootId: string) => void {
    return createSelector(
        (state, rootId) => getPostsForThread(state, {rootId}),
        () => {},
    );
}
```
you could remove the extra object if you wanted though